src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockAgoraDAO.scala

```diff
@@ -24,4 +24,8 @@ class MockAgoraDAO extends AgoraDAO {
     Future(List(MockAgoraDAO.agoraPermission))
   }
 
+  def status: Future[SubsystemStatus] = {
+    Future(throw new Exception("Agora Mock DAO exception"))
+  }
+
```

Comment:
Any reason MockAgoraDAO throws an exception for status, but other daos succeed? EDIT: ah, I see, you use this later in tests.